keps/prod-readiness/sig-cloud-provider/2395.yaml

```diff
@@ -1,3 +1,5 @@
 kep-number: 2395
 alpha:
   approver: "@wojtek-t"
+beta:
+  approver: "@johnbelamaric"
```

Comment:
@dims - we're generally trying to not change PRR approver in the middle if not necessary

Reply:
@wojtek-t will switch it back to you from John